src/application/snapshot/widgets/organization-proposals.container.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState } from 'react';
+
+import { useCommandQuery } from 'shared/messaging';
+import { Pagination } from 'shared/ui';
+
+import { GetProposalCommand } from '../commands';
+
+import { Proposal } from './proposal';
+
+interface Properties {
+  snapshotHandle: string;
+  className?: string;
+  top?: number;
+  onClose?: () => void;
+}
+
+export const OrganizationProposalsContainer = ({
+  snapshotHandle,
+  className = 'fixed top-20',
+  top,
+  onClose,
+}: Properties) => {
+  const [currentProposalIndex, setCurrentProposalIndex] = useState(0);
+
+  const proposalQuery = useCommandQuery({
+    command: new GetProposalCommand({
+      snapshotName: snapshotHandle ?? '',
+      pageNumber: currentProposalIndex,
+    }),
+    enabled: snapshotHandle ? snapshotHandle.length > 0 : false,
+    staleTime: Number.POSITIVE_INFINITY,
+    placeholderData: (previousData) => {
+      return previousData;
+    },
+  });
+
```

Comment:
I am wondering if it's still needed to check for snapshotHandle length etc. in typescript it seems like it's always a string, can you double check that please?

Reply:
You are right, we don't need that anymore